web/src/locales/en-US/cluster.js

```diff
@@ -347,4 +347,7 @@ export default {
   "cluster.providers.aliyun": "Aliyun",
   "cluster.providers.tencent-cloud": "Tencent-cloud",
   "cluster.providers.ecloud": "Ecloud",
+  
+  "cluster.metrics.dsl.copy": "Copy DSL",
```

Comment:
I would like to use `request` reather `dsl`

Reply:
Done.